core/client/app/services/clock.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    run
+} = Ember;
+
+const ONE_SECOND = 1000;
+
+// Creates a clock service to run intervals.
+
+export default Service.extend({
+    second: null,
+    minute: null,
+    hour:   null,
+
+    init() {
+        this.tick();
+    },
+
+    tick() {
+        let now = moment.utc();
+        this.setProperties({
+            second: now.seconds(),
+            minute: now.minutes(),
+            hour:   now.hours()
+        });
+
+        let self = this;
+        if (!Ember.testing) {
+            run.later(function() {
```

Comment:
@AileenCGN being _super_ nitpicky here, but could you change this to use the es6 syntax (just to keep it consistent):

```js
run.later(() => {
    this.tick();
}, ONE_SECOND);
```

this also means removing the `var self=this;` above.